package.json

```diff
@@ -209,15 +209,15 @@
     "react-native-mail": "4.1.0",
     "react-native-markdown-display": "7.0.0-alpha.2",
     "react-native-minimizer": "brunobar79/react-native-minimizer#26ae6dce10d353de701cebc3a1ae126729bf87f0",
-    "react-native-mmkv": "2.3.3",
+    "react-native-mmkv": "2.4.3",
```

Comment:
why this?

Reply:
I need [this](https://github.com/mrousavy/react-native-mmkv/commit/8af0d47c2e09e52e95e5ab240971ca0821568b0a) changes
I didn't fount wrong behavior but if it critical, i can make a patch